contracts/ERC20Decrypto.sol

```diff
@@ -550,4 +650,79 @@ contract ERC20Decrypto is
         _totalSupply = _totalSupply.sub(unformattedAmount);
         emit Transfer(account, address(0), amount);
     }
+
+    /**
+     * @dev Destroys `amounts` tokens from `accounts`, reducing the
+     * total supply.
+     *
+     * Emits a {Transfer} event with `to` set to the zero address.
+     *
+     * Requirements:
+     *
+     * - `accounts` cannot be the zero address.
+     * - `accounts` must have at least `amount` tokens.
+     */
+    function _burnBatch(
+        address sender,
+        address[] memory accounts,
+        uint256[] memory amounts
+    ) internal virtual {
+        require(
+            accounts.length == amounts.length,
+            "ERC20: accounts and amounts length mismatch"
+        );
+        for (uint256 i = 0; i < accounts.length; ++i) {
+            address account = accounts[i];
+            require(account != address(0), "ERC20: burn to the zero address");
+            _beforeTokenTransfer(account, address(0), amounts[i]);
+            uint256 unformattedAmount = _unformattedValue(amounts[i]);
+            _balances[account] = _balances[account].sub(
+                unformattedAmount,
+                "ERC20: burn amount exceeds balance"
+            );
+            _totalSupply = _totalSupply.sub(unformattedAmount);
+        }
+
+        address[] memory zeroAddress;
+        emit TransferBatch(sender, zeroAddress, amounts);
```

Comment:
```suggestion
        emit TransferBatch(accounts, zeroAddress, amounts);
```

Reply:
when you burn tokens the event is the sender, the address (0) [because you burned it] and the amounts it burns